### DISCUSSION TITLE
Setup - Distribution of Roles

### DISCUSSION THREAD

**Discussion by redblindbananaz** [The-Digital-Dilemma/GetIT#16] [General]
Open discussion of how we will distribute the backlog.
Please leave a comment for your preference

**Comment by redblindbananaz** [comment]
I'll be keen to do the models

**Comment by makikamiya** [comment]
I would like to have a go at getting the initial project structure coded. As mentioned in this morning's meeting, I'll create a separate branch for this work so that it can be reviewed before it is pulled into the master branch.

**Reply by redblindbananaz** [reply]
It would be good to have the apps created and registered, creation of basic models and setup templates\appname folders and static folders, let me know if there is anything I can do, I have lots of time in my hand. I will try to keep this Github project organised and up to date with the read me documents in the mean time. What would be your timeline for this? Maybe let's use the initialise stage for this so we are all working together in the basic stage? Let me know your thoughts, cheers
